shell_completions.md

```diff
@@ -48,25 +48,25 @@ Zsh:
   $ echo "autoload -U compinit; compinit" >> ~/.zshrc
 
   # To load completions for each session, execute once:
-  $ yourprogram completion zsh > "${fpath[1]}/_yourprogram"
+  $ %[1]s completion zsh > "${fpath[1]}/_yourprogram"
```

Comment:
We should also replace the second `yourprogram` in this line.

Reply:
Missed it, thanks!